tdrs-backend/tdpservice/search_indexes/admin/filter.py

```diff
@@ -0,0 +1,37 @@
+"""Filter classes."""
+from django.utils.translation import ugettext_lazy as _
+from django.contrib.admin import SimpleListFilter
+from tdpservice.data_files.models import DataFile
+
+class CreationDateFilter(SimpleListFilter):
+    """Simple filter class to show newest created datafile records."""
+
+    title = _('Newest')
+
+    parameter_name = 'created_at'
+
+    def lookups(self, request, model_admin):
+        """Available options in dropdown."""
+        return (
+            (None, _('Newest')),
+            ('all', _('All')),
+        )
+
+    def choices(self, cl):
+        """Update query string based on selection."""
+        for lookup, title in self.lookup_choices:
+            yield {
+                'selected': self.value() == lookup,
+                'query_string': cl.get_query_string({
+                    self.parameter_name: lookup,
+                }, []),
+                'display': title,
+            }
+
+    def queryset(self, request, queryset):
+        """Sort queryset to show latest records."""
+        if self.value() is None and len(queryset):
```

Comment:
For improved readability can we change this to be explicit on the length check? I almost wish we had a better way like `.exists()` or something to check. Nitpick overall and doesn't have to be changed

```suggestion
        if self.value() is None and len(queryset) > 0:
```